src/components/plots/FeaturesVsUMIsScatterplot.jsx

```diff
@@ -3,23 +3,29 @@ import PropTypes from 'prop-types';
 import { Vega } from 'react-vega';
 
 import EmptyPlot from './helpers/EmptyPlot';
+import transformOldFeaturesVsUMIsPlotData from './helpers/transformOldFeaturesVsUMIsPlotData';
 import generateSpec from '../../utils/plotSpecs/generateFeaturesVsUMIsScatterplot';
 
 const FeaturesVsUMIsScatterplot = (props) => {
   const {
     config, plotData, actions,
   } = props;
 
+  // can remove if we formally migrate
```

Comment:
Can you correct this comment to be more contextual? I'm afraid it'll confuse others. Maybe you can add something like "We can remove this if we migrate old plotdata for this plot to the new schema"

Reply:
done